server/tests/security/test_input_rail.py

```
@@ -0,0 +1,264 @@
+    _FAIL_CLOSED_REASON,
+    _INIT_FAILURE_BACKOFF_S,
+    check_input,
+)
+
+
+# ---------------------------------------------------------------------------
+# Helpers / fixtures
+# ---------------------------------------------------------------------------
+
+
+def _run(coro):
```

Ruff (0.15.12)

[warning] Missing return type annotation for private function `_run`

(ANN202)